src/SFA.DAS.AANHub.Api/Common/ActionResponseControllerBase.cs

```diff
@@ -33,6 +34,18 @@ public abstract class ActionResponseControllerBase : ControllerBase
         return new BadRequestObjectResult(FormatErrors(response.Errors));
     }
 
+    protected IActionResult GetPutResponse(ValidatedResponse<PutCommandResult> response, object? routeParameters)
+    {
+        if (response.IsValidResponse)
+        {
+            return response.Result.NewResourceCreated
+                ? new CreatedAtActionResult(PutMethodName, ControllerName, routeParameters, response.Result)
```

Comment:
CreatedAtAction should link to a GET endpoint. However, I think NoContent in any case is just fine.

Reply:
Replaced with NoContent